cluster-scope/bundles/curator-operator-system/kustomization.yaml

```diff
@@ -2,8 +2,18 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 
 resources:
+  - ../../base/apiextensions.k8s.io/customresourcedefinitions/curatorconfig.curatoroperator.io
+  - ../../base/apiextensions.k8s.io/customresourcedefinitions/fetchdata.curatoroperator.io
+  - ../../base/apiextensions.k8s.io/customresourcedefinitions/reports.api.curatoroperator.io
+  - ../../base/apiextensions.k8s.io/customresourcedefinitions/reports.curatoroperator.io
   - ../../base/core/namespaces/curator-operator-system
   - ../../base/rbac.authorization.k8s.io/clusterrolebindings/curator-operator-manager-rb
   - ../../base/rbac.authorization.k8s.io/clusterrolebindings/curator-operator-oauth-proxy-rb
   - ../../base/rbac.authorization.k8s.io/clusterroles/curator-auth-proxy-role
+  - ../../base/rbac.authorization.k8s.io/clusterroles/curator-operator-fetchdata-editor-role
+  - ../../base/rbac.authorization.k8s.io/clusterroles/curator-operator-fetchdata-viewer-role
   - ../../base/rbac.authorization.k8s.io/clusterroles/curator-operator-manager-role
+  - ../../base/rbac.authorization.k8s.io/clusterroles/curator-operator-report-api-editor-role
+  - ../../base/rbac.authorization.k8s.io/clusterroles/curator-operator-report-api-viewer-role
+  - ../../base/rbac.authorization.k8s.io/clusterroles/curator-operator-report-editor-role
+  - ../../base/rbac.authorization.k8s.io/clusterroles/curator-operator-report-viewer-role
```

Comment:
How are these 6 new cluterroles that you have added being used?

Reply:
Yes, All are being used by crds

Reply:
@skanthed I admit I don't know how to write CRDs, but I don't see any reference to these clusterroles in your CRDs. Though, I noticed a comment in those clusterroles that say they are for end users to edit/view reports etc. I assume those will be used later.

Reply:
Yes, will be used later. We don't specify that in crds. When I will deploy the controller when the PR is merged. It will look for roles before accessing crds.